setup.py

```diff
@@ -19,6 +19,7 @@ def build_native(spec):
 setup(
     name='slonik',
     version='0.0.1',
+    license='MIT',
```

Comment:
Actually, according https://packaging.python.org/tutorials/packaging-projects/, using trove classifier (`"License :: OSI Approved :: MIT License"`) is better suited to that.